MDANSE_GUI/Src/MDANSE_GUI/Tabs/Visualisers/PlotWidget.py

```diff
@@ -335,9 +353,23 @@ def make_canvas(self, width=12.0, height=9.0, dpi=100):
         normaliser.new_values.connect(self.normaliser_change)
         self._sliderpack = slider
         self._normaliser = normaliser
+        # New widgets, next to the toolbar
```

Comment:
Will this mean anything in 2 years?
```suggestion
        # Matplotlib configuration widgets, next to the toolbar
```